src/bpfilter/opts.c

```diff
@@ -33,6 +32,8 @@ static const char *_bf_verbose_strs[] = {
     [BF_VERBOSE_BYTECODE] = "bytecode",
 };
 
+const char *argp_program_version = "bpfilter version " BF_VERSION;
```

Comment:
In `bfcli/main.c`, the callback is set in the main function, this should be done similarly:

```
int main()
{
   ...

   bf_logger_setup();

   argp_program_version = "bpfilter version " BF_VERSION;

   ...
}
```